NashvilleHistory/www/js/controllers/AllPlacesCtrl.js

```diff
@@ -40,20 +51,36 @@ app.controller('AllPlacesCtrl', function($scope, $state, $q, AllPlacesFact){
 
   getAllPlaces();
 
+
+  function areMarkersBookmarked (){
+      BookmarkFact.getAllBookmarks(AuthFact.getUserId())
+      .then((bookmarks)=>{
+        console.log("bookmarked markers", bookmarks);
+        Object.keys(bookmarks).map((key)=>{
+          AllMarkers.forEach((marker, index)=>{
+            if (bookmarks[key].latitude === marker.latitude && bookmarks[key].longitude === marker.longitude){
```

Comment:
Maybe use something from the description of each marker as well. I think it might be better to agree on assigning a uid to each marker on retrieval, here's what I'm thinking rn on my Add To Route branch:
```
// Generate a UID property on each marker: First Word in Title + Marker Lat + Marker Long, strip periods and minuses
          if (AllMarkers[index].title) {
            AllMarkers[index].uid = AllMarkers[index].title.match(/^([\w\-]+)/)[0] + (AllMarkers[index].latitude + AllMarkers[index].longitude).replace(/\-|\./g,"")
          } else if (AllMarkers[index].artwork) {
            AllMarkers[index].uid = AllMarkers[index].artwork.match(/^([\w\-]+)/)[0] + (AllMarkers[index].latitude + AllMarkers[index].longitude).replace(/\-|\./g,"")
          }
```